.flake8

```diff
@@ -19,6 +19,7 @@ exclude =
     .eggs
 format = ${cyan}%(path)s${reset}:${yellow_bold}%(row)d${reset}:${green_bold}%(col)d${reset}: ${red_bold}%(code)s${reset} %(text)s
 ignore =
+    I100
```

Comment:
No, you use flake8 to fix what's wrong with the code, not to ignore mistakes

Reply:
Removed it again